etna/etna.gemspec

```diff
@@ -16,4 +16,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'net-http-persistent'
   spec.add_dependency 'multipart-post'
   spec.add_dependency 'rollbar'
+  spec.post_install_message = "\n\n\033[1;31mAdd 'source ~/etna.completion' to your .bashrc to get etna command line completions!\033[0m\n\n"
```

Comment:
Now, after installing with `gem install etna` it will provide a post install message instructing users to source the completion.

Reply:
Nice!